raiden_installer/tokens.py

```diff
@@ -208,6 +212,30 @@ class TokensV33(Enum):
     WIZ = _WizardToken
 
 
+class TokensV36(Enum):
+    RDN = Erc20Token(
```

Comment:
It might be better to read the token addresses from the respective smart contracts? the ServiceToken (RDN) you can always get from the user deposit contract

Reply:
Issue #104